spec/lib/krikri/enricher_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'spec_helper'
+
+describe Krikri::Enricher do
+
+  shared_context 'common' do
+    let(:chain) do
+      {
+        :'Krikri::Enrichments::StripHtml' => {
+          input_fields: [{sourceResource: :title}]
+        },
+        :'Krikri::Enrichments::StripWhitespace' => {
+          input_fields: [{sourceResource: :title}]
+        }
+      }
+    end
+    let(:mapping_activity_uri) do
+      (RDF::URI(Krikri::Settings['marmotta']['ldp']) /
+        Krikri::Settings['prov']['activity'] / '3').to_s
+    end
+  end
+
+  describe '#initialize' do
+    context 'with a chain argument that has been parsed from JSON' do
+      # Activity records will contain the serialized chain hash, which is
+      # passed in the call to Krikri::Enricher.enqueue.
+      include_context 'common'
+      let(:json_chain) do
+        s = <<-EOS.gsub /\s+/, ' '
```

Comment:
Yes.  I'm not sure what the context was, but I recall playing around with heredocs in `irb` and getting some kind of error that went away when I used an assignment.  I'll change this.